.github/workflows/cratesio-publish.yml

```diff
@@ -27,8 +27,23 @@ jobs:
           file: "Cargo.toml"
           key: "package.version"
           value: "${{ steps.get_version.outputs.VERSION }}"
+      
+      - name: Set the version for publishing on derive crate
```

Comment:
Probably need to also edit redismodule-rs-derive?

Reply:
This one is editing redismodule-rs-derive